### PR TITLE
Added network module

### PR DESCRIPTION
**Pull request by daveed12**
- Added hubblestack/modules/network.py
- Added hubblestack/utils/validate/net.py
- Added hubblestack/tests/unittests/modules/test_network.py

### CODE REVIEW

File: tests/unittests/modules/test_network.py [hunk 1]
```diff
@@ -0,0 +1,456 @@
+# -*- coding: utf-8 -*-
+
+# Import Python Libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+import os.path
+import socket
+
+import hubblestack.config
```
Comment (Tenebriso) [inline tests/unittests/modules/test_network.py:10]: It doesn't seem to find this package, I think you need to port this in https://github.com/saltstack/salt/tree/master/salt/config

**Review by Tenebriso — CHANGES_REQUESTED** [review]

File: tests/unittests/modules/test_network.py (hunk 1 above)
Comment (daveed12) [reply tests/unittests/modules/test_network.py:10]: Yea I believe Paul is working on that according to the wiki.

**Review by jettero — APPROVED** [review]

**Comment by jettero** [comment]
this seems to be waiting on #944

**Comment by jettero** [comment]
I really thought this was either merged or that I combined it into one of my big merges (like ubermerge).

We may need to get this synced by copying the files to /tmp, creating a new branch from develop, and copying these files back in (or soemthing similar). I'm not having any luck finding another way to merge these.

**Review by jettero — APPROVED** [review]

**Review by Tenebriso — APPROVED** [review]

**Comment by jettero** [comment]
(I forced the merge on this. develop doesn't pass tests with or without these commits... pretty sure anyway...)

**Pull request merged**